src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -86,6 +99,70 @@ public void handlesSocket() throws IOException {
     }
 
     /**
+     * BkBasic can handle HTTP persistent connections.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    @SuppressWarnings("PMD.DoNotUseThreads")
+    public void handlesPersistentConnection() throws Exception {
+        final int port = 8080;
+        final String uri = String.format("http://localhost:%d", port);
+        // @checkstyle MagicNumberCheck (1 line)
+        final int count = 1;
+        final CountDownLatch completed = new CountDownLatch(count);
+        final Take take = new Take() {
+            @Override
+            public Response act(final Request req) throws IOException {
+                return new TkEmpty().act(req);
+            }
+        };
+        new Thread(
+            // @checkstyle AnonInnerLengthCheck (23 lines)
+            new Runnable() {
+                @Override
+                public void run() {
+                    try {
+                        new FtBasic(
+                            new BkBasic(take),
+                                port
+                        ).start(
+                                new Exit() {
+                                    @Override
+                                    public boolean ready() {
+                                        return completed.getCount() == 0;
+                                    }
+                                }
+                        );
+                    } catch (final IOException ex) {
+                        throw new IllegalStateException(ex);
+                    }
+                }
+            }
+        ).start();
+        final HttpURLConnection conn = HttpURLConnection.class.cast(
+                new URL(uri).openConnection()
+        );
+        final String connection = "Connection";
+        final String keepAlive = "Keep-Alive";
+        conn.setRequestMethod("GET");
+        conn.setUseCaches(false);
+        conn.setInstanceFollowRedirects(false);
+        conn.addRequestProperty("Host", "localhost");
+        conn.addRequestProperty(connection, keepAlive);
+        MatcherAssert.assertThat(
+                conn.getHeaderFields(),
+                Matchers.hasKey(connection)
+        );
+        MatcherAssert.assertThat(
+                conn.getHeaderFields().get(connection),
+                Matchers.hasItem(keepAlive)
+        );
+        completed.countDown();
+        completed.await(1L, TimeUnit.MILLISECONDS);
+        MatcherAssert.assertThat(completed.getCount(), Matchers.equalTo(0L));
```

Comment:
@mstzn36 what exactly we check here? 
I think `completed.getCount()` always will be equals 0